railties/lib/rails/generators/rails/app/templates/test/system_test_helper.rb

```diff
@@ -0,0 +1,8 @@
+require "test_helper"
+
+class ActionSystemTestCase < ActionSystemTest::Base
```

Comment:
That feels a lot like generated configuration, on something I'd expect most people to just leave alone anyway. Am I underestimating how often applications will need to care about overriding this? :confused: